### PR TITLE
Create ruleset: Grsm.io

### PR DESCRIPTION
**Pull request by injust**

### CODE REVIEW

File: src/chrome/content/rules/Grsm.io.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+<ruleset name="Grsm.io">
+	<target host="grsm.io" />
+	<target host="*.grsm.io" />
+		<test url="http://www.grsm.io/" />
+		<test url="http://a.grsm.io/" />
+		<test url="http://b.grsm.io/" />
+
+	<rule from="^http://([\w-]+\.)?grsm\.io/"
+		to="https://$1grsm.io/" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Grsm.io.xml]: Keep it on one line.

File: src/chrome/content/rules/Grsm.io.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+		- grsm.io
+-->
+
+<ruleset name="Grsm.io">
+	<target host="grsm.io" />
+	<target host="*.grsm.io" />
+		<test url="http://www.grsm.io/" />
+		<test url="http://a.grsm.io/" />
+		<test url="http://b.grsm.io/" />
+
```
Comment (pipboy96) [inline src/chrome/content/rules/Grsm.io.xml]:
```suggestion
	<securecookie host=".+" name=".+" />
```

File: src/chrome/content/rules/Grsm.io.xml (hunk 1 above)
Comment (injust) [reply src/chrome/content/rules/Grsm.io.xml]: I prefer to only one-line `<rule>`s when it is a global redirect for all targets.

Comment (pipboy96) [reply src/chrome/content/rules/Grsm.io.xml]: @injust Got it.

File: src/chrome/content/rules/Grsm.io.xml (hunk 2 above)
Comment (injust) [reply src/chrome/content/rules/Grsm.io.xml]: Thanks, added.

**Pull request merged**